src/ontology/cl-edit.owl

```diff
@@ -31266,6 +31267,16 @@ AnnotationAssertion(rdfs:label obo:CL_4033036 "OFFx cell")
 SubClassOf(obo:CL_4033036 obo:CL_0000750)
 SubClassOf(obo:CL_4033036 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001786))
 
+# Class: obo:CL_4033037 (tracheobronchial mucus secreting glandular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30864819") obo:IAO_0000115 obo:CL_4033037 "A mucus secreting cell of a submucosal gland of the tracheobronchial tree.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033037 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033037 "2023-04-20T10:55:50Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30864819") oboInOwl:hasExactSynonym obo:CL_4033037 "mucus secreting cell of tracheobronchial tree submucosal gland")
+AnnotationAssertion(rdfs:label obo:CL_4033037 "tracheobronchial mucus secreting glandular cell")
+EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
```

Comment:
Based on definitions, it seems like the rdfs:label is too broad. If this class is purposefully meant to exclude goblet cells, consider using the synonym "mucus secreting cell of tracheobronchial tree submucosal gland" as the label instead.

Reply:
I like this and agree.

Reply:
Thank you both, the new label will also align with the other submucosal gland cells.